Justfile

```diff
@@ -38,5 +38,8 @@ calculate-artifact-url:
     checksum=$(bash scripts/ops/calculate-checksum.sh) && \
     echo "https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-$checksum.tar.gz"
 
+generate-monorepo-bindings:
```

Comment:
should we include this with generate-genesis? it seems like we would always want to run them together

Reply:
Didn't think it was too impactful yet since we don't have to run both in CI right now.

I can add a `generate-all` command that runs both though!

Reply:
yeah, that sounds good! I just think it's a potential footgun for a dev to forget to run one but not the other